### ISSUE TITLE
Section 3.3: correlation with HMISParticipationStatusStartDate and HMISParticipationStatusEndDate

### ISSUE THREAD

**Issue by nataliesus** [MollyMcEvilley/LSASampleCode#1251]
Hi @MollyMcEvilley ,

I want to confirm that project stay/bed-night:
- either on HMISParticipationStatusStartDate
- or on HMISParticipationStatusEndDate
are valid and should be counted if they are not negated with other restrictions, correct?

If yes, could you please re-review "<" and ">" operators in the Sample code and the Spec?

Here:
![image](https://github.com/user-attachments/assets/e2549453-6b00-4b90-9678-a95d28a58fe3)

and here:

![image](https://github.com/user-attachments/assets/7b0e8146-98de-457f-bbd6-dbcde8ed81ae)

Thank you!
Natalie

**Comment by MollyMcEvilley** [comment]
Dates = HMISParticipationStatusStartDate are valid and should be counted if not negated by other restrictions.
Dates (other than ExitDates) = HMISParticipationStatusEndDate are not counted. 

I've uploaded changes to Step 3.3.1 in the sample code -- leaving this open until I get through the specs.

I made this chart as a reference for myself as I'm going through all of the code -- feel free to ignore if not helpful.

![image](https://github.com/user-attachments/assets/7c981981-19a8-4e7e-af60-1782c5cc221c)
